binary/proto/scan_result.proto

```diff
@@ -89,6 +89,7 @@ message Inventory {
     SPDXPackageMetadata spdx_metadata = 14;
     JavaArchiveMetadata java_archive_metadata = 15;
     JavaLockfileMetadata java_lockfile_metadata = 31;
+    PACMANPackageMetadata pacman_metadata = 36;
```

Comment:
Run `make protos` to regenerate the `binary/proto/scan_result_go_proto/scan_result.pb.go` file